src/main/resources/db/migration/20170705125033654__initial_schema.sql

```diff
@@ -5,4 +5,18 @@
 CREATE TABLE widget (
     id uuid NOT NULL,
     name text NOT NULL
-);
\ No newline at end of file
+);
+
+--
+-- Name: sources_of_funds; Type: TABLE; Schema: buq; Owner: postgres; Tablespace:
+--
+
+CREATE TABLE sources_of_funds (
+    id uuid NOT NULL,
+    name text NOT NULL,
+    description text NOT NULL
+);
+
+CREATE UNIQUE INDEX unq_source_of_fund_name
+ON sources_of_funds (LOWER(name))
+WHERE name IS NOT NULL;
```

Comment:
is this migration added in this file on purpose? Shouldn't you have made it in a separate file?

Reply:
I did it on purpose. In the next task (adding CRUD) I will remove the part that adds the widget table. I wanted the first migration script on this service to be titled initial_schema.sql.